neurots/extract_input/input_parameters.py

```diff
@@ -24,74 +35,69 @@ def parameters(
     neurite_types=("basal", "apical", "axon"),
     feature="path_distances",
     diameter_parameters=None,
+    trunk_method="simple",
 ):
-    """Returns a default set of input parameters to be used as input for synthesis.
+    """Return a default set of input parameters to be used as input for synthesis.
 
     Args:
         origin (list[float]): The origin point.
         method (str): The method to use.
         neurite_types (list[str]): The neurite types to consider.
         feature (str): Use the specified TMD feature.
         diameter_parameters (dict or str): The parameters used for the diameters.
+        trunk_method (str): 'simple' for simple trunk method, or '3d_angles'.
```

Comment:
to remove and append by default

Reply:
I cannot have both in the parameters, as this is where we decide which method we want to use